### DISCUSSION TITLE
View Collections

### DISCUSSION THREAD

**Discussion by Techthy** [pocketbase/pocketbase#3920] [Q&A] [answered]
Hi so I am trying to have a view collection where I merge a boolean column together. Is there a way to tell pocketbase again that it is a boolean in deed?
Here my query (vegetarian exists both on regiokon and users)
```
SELECT 
  regiokon.id as id, regiokon.paid,
  CAST(COALESCE((users.name || ' ' || users.surname), regiokon.name) AS TEXT) as name,
  CAST(COALESCE(users.email, regiokon.email) AS TEXT) as email,
  CAST(COALESCE(users.phonenumber, regiokon.phonenumber) AS TEXT) as phonenumber,
  regiokon.needs_lodging,
  CAST((users.allergies || regiokon.allergies) AS TEXT) as allergies, 
  COALESCE(users.vegetarian, regiokon.vegetarian) AS vegetarian
  from regiokon LEFT JOIN users on regiokon.user = users.id ORDER BY name 

```

rn I am getting numbers 0 and 1 :)

**Comment by ganigeorgiev** [comment, marked as answer]
Using `CAST(yourColumnOrExpr as bool) as fieldName` should work.

_More finer controls over the generated view schema fields will be available in the future._

**Reply by Techthy** [reply]
Many thanks. Maybe I got thrown off by the fact that bool is not highlighted and also not available as a tab completion :)

**Reply by ganigeorgiev** [reply]
_I'll update the syntax highligher and keywords completion with the next release._
